app/main/view.ts

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+import { BrowserView, BrowserWindow, app } from 'electron';
+
+import ConfigUtil = require('../renderer/js/utils/config-util');
+const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
```

Comment:
Should we set a backup value for this? In case the json gets corrupted? i.e.
```
const shouldSilentWebview = ConfigUtil.getConfigItem('silent') || false;
```

Reply:
I picked it up from webview.ts . Is this necessary? If yes, then i'll change it.

Reply:
Leave it for now then.